dev/image-guidelines.md

```diff
@@ -0,0 +1,329 @@
+
+![](image-guidelines/excalidraw_annotations.png)
+*Examples of arrows and numbered callouts*
+
+The above example also shows that a semi-transparent rectangle can be placed behind text to make it more legible if it overlays a dark area of background.
+#### Excalidraw-specific options
+
+One of the hidden benefits of Excalidraw is that it doesn't have too many settings.
+This automatically leads to greater consistency.
+
+For the line "**sloppiness**" setting we use the **"medium"** option, i.e. "artistic". The "cartoonist" option produces divergent lines and in general is a bit too messy. We want humanized, not messy (even though messy is fun!).
+
+For line thickness ("**stroke width**") we use "**extra bold**" for annotation arrows, but this is subjective choice, since line thickness is ultimately relative to the dimensions of the rendered image.
+The "extra bold" line thickness is what we want to achieve when the screenshot on the canvas is at an approximate 1:1 ratio (i.e. sized similar to the orginal source) unzoomed.
+
+If you want really thick lines or huge fonts then overlay them on a small (resized) background image. 🙂
+
+
+![](image-guidelines/excalidraw_defaults.png)
+
+
+
+## Line diagrams
+
+We also use Excalidraw for line diagrams. These can be drawn from scratch but Excalidraw also allows importing some Mermaid markdown which can be useful for quickly creating initial diagram structures.
+
+The following image highlights the recommended Excalidraw settings for our diagrams. Generally these are similar to the settings for annotations but note the following differences:
+
+- stroke width is **Bold** (not Extra Bold)
+- stroke color is black (not blue)
+
+![](image-guidelines/line_diagram_settings.png)
+
+The diagram heading is made bold by choosing our #2 font which is the same font as our #1 font but a bold variant. This is necessary because Excalidraw doesn't have any options for bolding or italicizing fonts.
+
+## Exporting images
+
+### Formats and settings
+
+When exporting images from Excalidraw we include "Background" which encodes the screenshot (if there is one) plus a small margin around it. For line diagrams this option creates an opaque white background plus a small margin.
+
+The Scale setting is a bit confusing:
+
+- If you've inserted a screenshot and not resized it, then export it to PNG with Scale=1x, this will produce an image that is about 1/2 the size of the original. For that reason either resize the image on the canvas to approximately it's original size or else choosing Scale=2x for the export will produce an approximate 1:1 of the original size. In other words the Scale setting is relative to the size of the image on your screen.
+- For SVG exports the Scale setting is irrelevant: it will base64-encode the screenshot at it's original size, even if you've scaled the screenshot on the Excalidraw canvas.
```

Comment:
Lets just standardise on svg - there is no point messing with scaling here and we want to reduce decision friction

Reply:
Yes agreed. I think that's what I meant to say but wrote that before I had come to that realisation.